backend/pong/login/views/totp.py

```
@@ -0,0 +1,176 @@
+                    utils.OpenApiExample(
+                        "Example 500 response (予期せぬエラーの場合)",
+                        value={
+                            "status": "error",
+                            "code": "internal_error",
+                        },
+                    ),
+                ],
+            ),
+        },
+    )
+    def POST(self, request: request.Request) -> response.Response:
```

Ruff (0.8.2)

Function name `POST` should be lowercase

(N802)